src/Coordination/KeeperStateMachine.cpp

```diff
@@ -153,6 +155,14 @@ void assertDigest(
 
 }
 
+std::unique_lock<std::mutex> KeeperStateMachine::getStorageLock() const
```

Comment:
I think we can avoid removing TSA annotations if we implement a lock_guard like wrapper instead of this function:
```
       struct TSA_SCOPED_LOCKABLE LockGuardWithStats final
        {
            std::unique_lock<std::mutex> lock;
            explicit LockGuardWithStats(std::mutex & mutex) TSA_ACQUIRE(mutex)
            {
                Stopwatch watch;
                std::unique_lock l(mutex);
                ProfileEvents::increment(ProfileEvents::KeeperStorageLockWaitMicroseconds, watch.elapsedMicroseconds());
                lock = std::move(l);
            }
            ~LockGuardWithStats() TSA_RELEASE() = default;
        };
```